pb-rs/src/parser.rs

```diff
@@ -60,7 +60,7 @@ fn integer(input: &str) -> IResult<&str, i32> {
 fn comment(input: &str) -> IResult<&str, ()> {
     value(
         (),
-        tuple((tag("//"), many0(not(line_ending)), opt(line_ending))),
+        tuple((tag("//"), take_while(|c| !"\n\r".contains(c)), opt(line_ending))),
```

Comment:
Just personal preference, but I would remove `opt(line_ending)`; would like to separate comment parsing from newline parsing.